src/emu/epoc/src/services/fbs/impls/bitmap.cpp

```diff
@@ -544,6 +588,45 @@ namespace eka2l1 {
         ctx->set_request_status(epoc::error_none);
     }
 
+    void fbscli::resize_bitmap(service::ipc_context *ctx) {
+        const auto fbss = server<fbs_server>();
+        const epoc::handle handle = *(ctx->get_arg<std::uint32_t>(0));
+        fbsbitmap *bmp = fbss->get<fbsbitmap>(handle);
+        if (!bmp) {
+            ctx->set_request_status(epoc::error_bad_handle);
+            return;
+        }
+
+        while (bmp->clean_bitmap != nullptr) {
```

Comment:
We can make explicit function to get clean bitmap

Reply:
Most of the things look good other than this, since i think we also duplicate this code in opcode implement function get_clean_bitmap

Reply:
function get_clean_bitmap is extracted